src/Management/ResourceBuilder.php

```diff
@@ -410,6 +412,14 @@ private function buildUser(array $data): User
         ]);
     }
 
+    private function buildOrganization(array $data): Organization
+    {
+        return $this->createObject(Organization::class, [
+            'name' => $data['name'],
+            'sys' => $this->buildSystemProperties($data['sys'])
```

Comment:
neither is consistent :( Sys is all over the place and some things have a trailing comma and some don't